prac_05/hex_colours.py

```diff
@@ -1,3 +1,20 @@
 COLOUR_CODE = {"maroon": "#b03060", "orchid": "#da70d6", "pink": "#ffc0cb", "salmon": "#fa8072",
                 "thistle": "#d8bfd8", "violet": "#ee82ee", "wheat": "#f5deb3", "linen": "#faf0e6",
-                "lavender": "#e6e6fa", "brown": "#a52a2a"}
\ No newline at end of file
+                "lavender": "#e6e6fa", "brown": "#a52a2a"}
+def main():
+    print(f"Colours available to print are ", end='')
+    for i in COLOUR_CODE:
+        if i != "brown":
+            print(f"{i}, ", end='')
+    print("and brown")
+    colour_choice = input("Please enter your colour choice: ").lower()
+    while colour_choice != "":
+        print_colour_code(colour_choice)
+        colour_choice = input("Please enter your colour choice: ").lower()
+    exit()
+
```

Comment:
You dont actually need a function for this, you could do a while loop.
eg. 
while colour_name = "":
        if colour_name in colour_code :
           print(
        else
           error

Reply:
implemented feedback https://github.com/IanEllimanJCU/cp1404practicals/commit/e06acba3aa52b11c868df270e67f6b48be56d66b#diff-6ce6ac9b49391884044badb19ebb179c